WordPress/Classes/Services/NotificationSyncMediator.swift

```diff
@@ -270,6 +312,23 @@ class NotificationSyncMediator {
             self.contextManager.save(derivedContext)
         }
     }
+
+    /// Invalidates the local cache for all the notifications with specified ID's in the array.
+    ///
+    func invalidateCacheForNotifications(with noteIDs: [String]) {
```

Comment:
Same here - we can reduce the redundancy by having the original `invalidateCacheForNotification` method call this new one.